goobi_adapter/terraform/topics.tf

```diff
@@ -0,0 +1,28 @@
+data "aws_iam_policy_document" "publish_to_topic" {
```

Comment:
Our SNS module provides a publish policy output.

Reply:
yeah, but we can't use that for SNS bucket notification cause you need the condition on the bucket ARN in the policy